content/developer/reference/backend/views.rst

```diff
@@ -178,6 +178,13 @@ an impact on all view types.
 
 .. todo:: Views main content section, with field, group & separator ?
 
+* ``class``
+  optional string containing some classes to set on the root node of the view.
+
+* ``js_class``
+  optional view registry key determining the component class to use to render the view.
+  If set, the class ``o_'js_class'_view`` will be set on the root node of the view.
+
```

Comment:
would like to add a link to "Customizing an existing view"

Reply:
I don't find this page in the documentation, which one is it ?

Reply:
section of javascript_cheatsheet.rst

Reply:
not updated. not a good idea